fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/FedoraTransaction.java

```diff
@@ -73,6 +72,23 @@ public void keepAlive() {
 		repo.keepTransactionAlive(txUri);
 	}
 	
+	public void cancel(Throwable t) {
```

Comment:
The two cancel methods are basically the same except for the rethrowing bit.  Since cancelling actually involves a fair number of steps, maybe we could refactor this so that cancel() just calls cancel(t), passing it null for the exception?